packages/@glimmer/vm-babel-plugins/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@glimmer/vm-babel-plugins",
+  "version": "0.73.2",
+  "description": "Compiles out VM assertion and deprecation utilities and debug tooling based on environment",
+  "repository": "https://github.com/glimmerjs/glimmer.js",
+  "author": "Tom Dale <tom@tomdale.net>",
+  "license": "MIT",
+  "private": false,
+  "scripts": {
+    "test": "mocha -r esm"
+  },
+  "dependencies": {
+    "babel-plugin-debug-macros": "^0.3.3"
```

Comment:
Will want to bump to `^0.3.4` now

https://github.com/ember-cli/babel-plugin-debug-macros/releases/tag/v0.3.4